### DISCUSSION TITLE
Is it possible to use UI.error, UI.success, ...  in the FastlaneSwiftRunner project?

### DISCUSSION THREAD

**Discussion by ddeveloperuch** [fastlane/fastlane#21804] [General]
Is it possible to use **UI.error, UI.success, ..**.  in the _FastlaneSwiftRunner_ project?

**Comment by ddeveloperuch** [comment]
About 5 months ago I asked a question, maybe someone knows something about this topic?

**Comment by florence0239** [comment]
Hello,

Yes, you can use UI.error, UI.success, and other logging methods in a Fastlane SwiftRunner project. These methods are part of Fastlane’s UI module and provide logging capabilities for different message types.

Here's how to use them in your Fastfile.swift:

import Foundation
import Fastlane

func customLane() {
    UI.message("This is a standard message")
    UI.success("This is a success message")
    UI.error("This is an error message")
    UI.important("This is an important message")
    UI.verbose("This is a verbose message")
    UI.command("This is a command message")
}

defaultPlatform(.ios)

platform :ios do {
    desc("Custom lane with logging")
    lane :customLane {
        customLane()
    }
}

Ensure you import the Fastlane module and then use UI.message, UI.success, UI.error, and other methods to log messages. Run the lane using fastlane ios customLane to see the messages in the console. This helps in clearly displaying various types of messages during your CI/CD process.

Best Regard,
florence023
[gm-socrates](https://www.gm-socrates.com)

**Comment by ddeveloperuch** [comment]
@florence0239 Do I need to add an extra package for this because I have an issue with the **Fastlane** import:
![fastlaneImport_issue](https://github.com/fastlane/fastlane/assets/5438527/65728c7d-3238-433a-97b2-6dd336f3cafa)
